vercel.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": 2,
+  "env": {
+    "GA_ID": "UA-543420-21"
```

Comment:
Let’s remove Google Analytics since it is a privacy bad practice

Reply:
Sure! I didn't want to edit existing code (this tracking already existed), but I can remove this.

Reply:
Removed GA.